tests/unit/amo/components/TestCollection.js

```diff
@@ -765,7 +750,7 @@ describe(__filename, () => {
     }));
 
     const wrapper = renderComponent({ store });
-    expect(wrapper.find('.Collection-edit-link')).toHaveLength(1);
+    expect(wrapper.find('.Collection-edit-link')).toHaveLength(0);
```

Comment:
We don't want to change this test. The edit link *should* remain visible for a collection owner, and only be absent if the user is not the owner of the collection.